tests/viz/test_edge_connections.py

```
@@ -0,0 +1,717 @@
+        graph = make_outer()
+        nodes, edges = extract_geometries(page, graph, depth=2)
+
+        validator = EdgeConnectionValidator(nodes, edges, tolerance=0.0)
+        issues = validator.validate_all()
+
+        assert issues == {}, f"Edge connection issues:\n{format_issues(issues)}"
+
+    def test_outer_depth2_structure(self, page):
+        """Fully expanded should show all internal nodes."""
+        graph = make_outer()
+        nodes, edges = extract_geometries(page, graph, depth=2)
```

Ruff (0.14.14)

[warning] Unpacked variable `edges` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)